docs/troubleshooting/node_export.md

```diff
@@ -0,0 +1,29 @@
+# Report a Node Bug
+
+When it comes to node issues, it is best to export specific information, which can help us debug and help with the problem
+
+## Export node json
+
+Each node hass an Export button, this will export all data of this specific node. To perform this export, use our screen shots and steps as your guide.
+
+### Select node from your webinterface
```

Comment:
I prefer to use a numbered list here:

1. Select node from web interfcace
....
2.
....